packages/playground/src/App.vue

```diff
@@ -1,10 +1,28 @@
 <script setup lang="ts">
 import { Node, Input } from './components';
-import { EditorDocument, EditorJSModel } from '@editorjs/model';
-import { data } from '@editorjs/model/dist/mocks/data.js';
+import { EditorDocument, EditorJSModel, EventType } from '@editorjs/model';
+// import { data } from '@editorjs/model/dist/mocks/data.js';
+import { ref } from 'vue';
 
-const document = new EditorDocument(data);
-const model = new EditorJSModel(data);
+const model = new EditorJSModel({
+  blocks: [ {
+    name: 'paragraph',
+    data: {
+      text: {
+        value: '',
+        $t: 't',
+      },
+    },
+  } ],
+});
+const document = ref(new EditorDocument(model.serialized));
+
+const serialized = ref(model.serialized);
+
+model.addEventListener(EventType.Changed, () => {
+  serialized.value = model.serialized;
+  document.value = new EditorDocument(model.serialized);
+});
```

Comment:
look strange, that you're getting information using existing `model.serialized`, but not from the Event you're listening. Is it the best way?

Reply:
Event has only the bit of info which was changed (e.g. some data or block) so we need to have the full object from the model